bin/hhBtagTest.cc

```diff
@@ -0,0 +1,27 @@
+#include "../interface/HH_BTag.h"
+
+int main()
+{
+    hh_btag::HH_BTag test("/afs/cern.ch/work/m/mdidomen/hh-italian/Analysis_2018/CMSSW_10_2_16/src/hh-bbtautau/Studies/python/prova_pipo_v2_par1.pb");
+
+    std::vector<float> jet_pt = {60, 50, 60};
```

Comment:
put numbers for some real event, so that you can check that the output is the same for c++ and python implementation.